### DISCUSSION TITLE
Using composer vimeo in admin view

### DISCUSSION THREAD

**Discussion by vfxhost** [joomengine/Joomla-Component-Builder#625] [Q&A] [answered]
Hello!
I'm moving my question from the old forum here, since I had no answer.
I use  Admin View custom PHP postSaveHook Method to get thumbnail_url from vimeo based on vimeo_video_url field value and store that information to the database.
To get the data from Vimeo I am using composer.
So, I need to add the following to the top of controllers/myitemview.php, manually to make my code to  work.
**require JPATH_SITE.'/vendor/autoload.php';
use Vimeo\Vimeo;**
Is there any way to do it from JCB?
![public_html_administrator_components_com_documentaries_controllers_title_php](https://user-images.githubusercontent.com/75916825/102710276-e6977280-42b9-11eb-8a5a-4a9827ea5c3d.jpg)

**Comment by Llewellynvdm** [comment, marked as answer]
We have been working on a pro/feature to give access to update the headers of the model and controller per/admin view. We will then also target all files eventually. This release should come out this week hopefully. So if you are not already a pro/member then [subscribe](https://vdm.bz/get-jcb-pro-membership) and get early access to pro/features.
